src/SimpleAmqpClient/Channel.h

```diff
@@ -91,6 +91,30 @@ class SIMPLEAMQPCLIENT_EXPORT Channel : boost::noncopyable {
                                        frame_max);
   }
 
+  /**
+    * Creates a new channel object
+    * Creates a new connection to an AMQP broker using the supplied parameters
+    * and opens a single channel for use
+    * This channel uses the EXTERNAL SASL method for authentication to the broker
+    * @param host The hostname or IP address of the AMQP broker
+    * @param port The port to connect to the AMQP broker on
+    * @param username The username used to authenticate with the AMQP broker
+    * @param vhost The virtual host on the AMQP we should connect to
+    * @param channel_max Request that the server limit the number of channels
+    * for
+    * this connection to the specified parameter, a value of zero will use the
+    * broker-supplied value
+    * @param frame_max Request that the server limit the maximum size of any
+    * frame to this value
+    * @return a new Channel object pointer
+    */
+  static ptr_t CreateSaslExternal(const std::string &host = "127.0.0.1", int port = 5672,
+                      const std::string &username = "guest",
```

Comment:
To be consistent with rabbitmq-c API, this should be called 'identity'.

Reply:
Done